source/features/discussion-links-on-repo-lists.tsx

```diff
@@ -20,7 +20,7 @@ function init(): void {
 		select('[href*="issues?q=label%3A%22help+wanted"]', repository)?.remove();
 
 		// Place before the "Updated on" element
-		select('relative-time', repository)!.parentElement!.before(
+		select('relative-time', repository)!.previousSibling!.before(
```

Comment:
parent element on work on the search tab not the user profile tab.

![image](https://user-images.githubusercontent.com/16872793/79069810-3a6ba980-7c9f-11ea-8e22-121906733c39.png)

Reply:
It looks like the DOM is different:

- global search wraps the time: https://github.com/search?q=user%3Asindresorhus+is

<img width="272" alt="global" src="https://user-images.githubusercontent.com/1402241/79069884-0eb5e700-7cd2-11ea-9188-08c94008e296.png">

- user profile doesn't: https://github.com/sindresorhus?tab=repositories&type=source

<img width="276" alt="user profile" src="https://user-images.githubusercontent.com/1402241/79069892-21302080-7cd2-11ea-8d11-f68063b5c2b5.png">

`previousSibling` produces the wrong DOM in the global search, but maybe it's fine. The alternative may be too wordy.